pkg/app/server/service/apiservice/service.proto

```diff
@@ -150,6 +151,17 @@ message DisableApplicationResponse {
     string application_id = 1 [(validate.rules).string.min_len = 1];
 }
 
+message UpdateApplicationRequest {
+    string application_id = 1 [(validate.rules).string.min_len = 1];
+    string piped_id = 2 [(validate.rules).string.min_len = 1];
```

Comment:
This field is not being used in the current implementation of UpdateApplication rpc, right?
If you want to update that field as well, it's okay to make it. Also, I read through the current implementation of UpdateApplication rpc, and it looks like it does exactly the same as we do in web_api.go (ref [code](https://github.com/pipe-cd/pipecd/blob/master/pkg/app/server/grpcapi/web_api.go#L530-L551)). So how about updating it like that? (Of course we have to keep the API key check stub for the client api.go version)

Reply:
Thanks for good review. `piped_id` can be referenced from application, so this field doesn't need and I improve the implementation of UpdateApplication rpc.